src/components/ExternalLink/__tests__/ExternalLink.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import ExternalLink from '../index';
+import * as trackModule from '../../../utils/track';
+import userEvent from '@testing-library/user-event';
+
+describe('ExternalLink', () => {
+  const component = (
+    <ExternalLink href="https://www.amazon.com" className="footer-link">
+      Click Here!
+    </ExternalLink>
+  );
+
+  it('should render the ExternalLink component', async () => {
+    render(component);
+    const externalLink = await screen.findByText('Click Here!');
```

Comment:
Can we make this a little more specific; maybe findByRole('link') (or getByRole? I always forget which to use) and test that it has the appropriate attributes? i.e. an external link should have attributes `rel="noopener noreferrer"` and `target="_blank"`